cmd/server/server.go

```diff
@@ -178,6 +178,14 @@ func (fac *backupRepoFactory) New(projectSpec models.ProjectSpec, storer models.
 	return postgres.NewBackupRepository(fac.db, projectSpec, storer)
 }
 
+type jobDependencyRepoFactory struct {
```

Comment:
Can we get rid of repoFactory and use the repo directly.

Reply:
removed